tests/framework/extensions/hardening/k3s/harden_nodes.go

```diff
@@ -10,29 +10,33 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func HardeningNodes(client *rancher.Client, hardened bool, nodes []*nodes.Node, nodeRoles []string) error {
+const (
+	HardenedKubeVersion = "v1.24.99"
```

Comment:
same nit as before, but also since these are public Const you should only define them in one place, and import it from there everywhere else you'll use it.

Reply:
Not sure I follow? This is only being used in this file. Before the refactor, it was in `config.go`, but that was due to it being used as a conditional for v1.25+ not being supported. Now that it's supported, we don't need that conditional anymore

Reply:
You're right, I forgot it was over there as part of this change 😅 . I can fix that